2023/01.md

```diff
@@ -87,6 +87,8 @@ Supporting materials includes slides, a link to the proposal repository, a link
     | ✓ | stage | timebox | topic | presenter |
     |:-:|:-----:|:-------:|-------|-----------|
     | ✓ | 3     | 30m     | [Intl Locale Info API](https://github.com/tc39/proposal-intl-locale-info)  Stage 3 update (Slides TBW, [spec](https://tc39.es/proposal-intl-locale-info)) | Frank Yung-Fong Tang |
+    |   | 3     | 30m     | [Explicit Resource Management](https://github.com/tc39/proposal-explicit-resource-management/) Stage 3 update (slides TBD, [spec](https://tc39.es/proposal-explicit-resource-management/)) |
```

Comment:
hopefully this can be discussed either right after or before my "Discuss SuppressedError argument overlap" item.